sys/include/byteorder.h

```diff
@@ -27,6 +27,10 @@
 #   include "clang_compat.h"
 #endif
 
+#if defined (MODULE_MYNEWT_CORE)
```

Comment:
is there a way to not include this here? feels like bad style to have a special case for a package.

Reply:
I can see if it can be patched in `mynewt-core`